Base/QTApp/qSlicerMainWindow.cxx

```diff
@@ -1471,8 +1471,8 @@ void qSlicerMainWindow::onFileSaved(const qSlicerIO::IOProperties& filePropertie
     // which can cause complication when attempted to be stored,
     // therefore we create a new clean property set.
     qSlicerIO::IOProperties properties;
-    properties["fileName"] = fileName;
-    properties["fileType"] = QString("SceneFile");
+    properties.insert("fileName", fileName);
+    properties.insert("fileType", QString("SceneFile"));
```

Comment:
Since the use of `insert()` will add duplicated entry, we should revisit to use `replace()`.

> If there is already an item with the same key in the map, this function will simply create a new one. (This behavior is different from replace(), which overwrites the value of an existing item.)

Source:
* https://doc.qt.io/archives/qt-5.15/qmultimap.html#insert
* https://doc.qt.io/qt-6/qmultimap.html#insert-1